internal/core/tests.rs

```diff
@@ -57,33 +57,54 @@ pub extern "C" fn slint_send_mouse_click(
     );
 }
 
+/// Simulate a character input event (pressed or released).
+#[no_mangle]
+pub extern "C" fn send_keyboard_char(
```

Comment:
Yes we need it because `send_keyboard_sequence` sends `press` and `release` but I need to do an `press` for a modifier and `release` on a specific place.

Reply:
Renamed it to _slint